MicroProfile-JWT-Auth/pom.xml

```diff
@@ -81,7 +84,7 @@
         <dependency>
             <groupId>fish.payara.microprofile.jwt-auth</groupId>
             <artifactId>microprofile-jwt-auth</artifactId>
-            <version>${payara.version}</version>
+            <version>${payara.build.version}</version>
```

Comment:
How is this any different? You're just creating a second property with (by default - and with no visible override) the same value as the `payara.version` property.

Reply:
This allows you to build with `-Dpayara.version=5.191.RC1 -Dpayara.build.version=5.191`, since that's the actual version of the JWT module

Reply:
That does add some complexity. Is it possible to strip any suffixes from the version instead?

Reply:
Unfortunately not. You can set properties in that fashion with the `build-helper-maven-plugin`, but profiles can't be activated from these properties. The only way would be through an extension to my profile activator plugin. Should I do that?